.changeset/strange-snakes-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/dev-utils': patch
```

Comment:
```suggestion
'@backstage/dev-utils': minor
```

Switching to a minor bump since this is a 1.0 package and there's new api/features.